api/client.go

```diff
@@ -43,6 +43,15 @@ type Config struct {
 	// RoundTripper is used by the Client to drive HTTP requests. If not
 	// provided, DefaultRoundTripper will be used.
 	RoundTripper http.RoundTripper
```

Comment:
The roundtripper is where you inject any authentication or any credentials you want to set for the requests.

Reply:
oh, you're right. I didn't realize about this one.
Thanks for the heads up!

Reply:
No problem! :)